### ISSUE TITLE
Ensure everything has proper landmarks out of the box

### ISSUE THREAD

**Issue by sarahmonster** [octopusthink/ghost-ship#21]
`main`, main should have ID of content

**Comment by sarahmonster** [comment]
Looks like this was done with #24, so we can close! 🎉
